### PR TITLE
Added details on HttpVNext Gateway Channel

### PR DESCRIPTION
**Pull request by welshdave**
Created documentation for HttpVNext Gateway Channel, as requested by @andreasohlund (https://github.com/welshdave/NServiceBus.Gateway.Channels.HttpVNext/issues/1).

### CODE REVIEW

File: components/components.yaml [hunk 1]
```diff
@@ -938,4 +938,15 @@
...
+  Key: GatewayChannelHttpVNext
+  Name: HttpVNext Gateway Channel
+  DocoUrl: nservicebus/gateway/httpvnext-channel
+  GitHubOwner: welshdave
+  Category: Other
+  SupportLevel: Community
+  LicenseUrl: https://github.com/welshdave/NServiceBus.Gateway.Channels.HttpVNext/blob/master/LICENSE
+  ProjectUrl: https://github.com/welshdave/NServiceBus.Gateway.Channels.HttpVNext
+  NugetOrder:
+    - NServiceBus.Gateway.Channels.HttpVNext
```
Comment (andreasohlund) [inline components/components.yaml]: @Particular/docs-maintainers the build server blow up with 

`System.Exception : Could not find nuget for id:NServiceBus.Gateway.Channels.HttpVNext`

but the nuget name seems correct to me? (https://www.nuget.org/packages/NServiceBus.Gateway.Channels.HttpVNext)

**Comment by SzymonPobiega** [comment]
I fixed the problem with the nuget (missing alias) but now it complaints again. I believe this would be resolved by rebasing the PR. @andreasohlund feel free to merge it, should be fine. Otherwise we'll fix it after the merge.

**Comment by andreasohlund** [comment]
@welshdave would you mind rebasing this?

**Comment by welshdave** [comment]
@andreasohlund done

**Comment by SzymonPobiega** [comment]
It turned out to be not related to the rebase but rather a missing `-` that I did not catch. Re-running the build and I'll merge once green. Thanks @welshdave !

**Pull request merged**